e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/deals/dashboard-table-data.spec.js

```diff
@@ -76,19 +75,19 @@ context('View dashboard deals as a maker', () => {
     // first deal should be the most recently updated (with our test data - GEF)
     //---------------------------------------------------------------
 
-    cy.get('table tr').eq(1).as('firstRow').find(`[data-cy="deal__status--${gefDeal._id}"]`).should('exist');
-
-    cy.assertText(exporter(gefDealId), gefDeal.exporter.companyName);
+    cy.assertText(dashboardDeals.rowIndex.exporter(), GEF_DEAL_BANK_2_MAKER_2.exporter.companyName);
 
-    cy.assertText(bankRef(gefDealId), gefDeal.bankInternalRefName);
+    // cy.assertText(bankRef(), gefDeal.bankInternalRefName);
+    cy.assertText(dashboardDeals.rowIndex.bankRef(2), GEF_DEAL_BANK_2_MAKER_2.bankInternalRefName);
 
-    cy.assertText(product(gefDealId), gefDeal.dealType);
+    cy.assertText(dashboardDeals.rowIndex.product(), CONSTANTS.DEALS.DEAL_TYPE.GEF);
```

Comment:
should this have a row?

Reply:
The test fails if row is used as it is unable to locate the deal on the dashboard page.